manim/utils/cfg_file_utils.py

```diff
@@ -67,9 +69,8 @@ def replace_keys(default):
     return default
 
 
-def main():
+def write(level=None):
```

Comment:
Suggestion: I think it'd be great to add a few more config options that have nothing to do with logging colors to this function in this PR.

Reply:
Is it really necessary in _this_ PR ? 
Adding more stuff that can be modified by the `write` command won't be pretty hard in and of itself, but we'd need to do input validation on each and every thing the user will try and input. 

The `logging` section's input validation was pretty straightforward, @naveen521kk just fed it through rich's own `Style.parse` method and it checked whether the style was valid.

It'll be a bit harder to perform input validation on the other stuff, especially since the configparser stores them as strings and we can't even check to see if the datatypes will match.

While it _is_ possible, I personally think that it'll deviate from the purpose of this PR, which is to get a subcommand system up and running.
Of course, I'll go ahead with doing all the input validation stuff if you insist, but it'll take more time than I'd like before this PR can be merged.

Reply:
The point is that if a user were to run the current `cfg write` command, they might the impression that it is only intended to change/write/modify logging-related configurations, and not all of the rest.

I do see your point about input validation and I do acknowledge that might get tricky.

How about, for this PR we add one more config option that has nothing to do with logging/rich/colors, and that is a Boolean? That way the input validation is fairly straightforward and we'll leave a blueprint for how to add more in the future. And the `cfg write` subcommand won't have only rich/color configs.

Reply:
Alright, so I've added the ability to edit just those configs whose values are booleans in the latest commit. There's still a lot of room for improvement in the input validation for it, and code reusability and all that, but I think another Issue/PR pair should be the place to discuss that.

For now though, you can set stuff like `write_to_movie` and whatnot.